content/blog/posts/graduation-2016

```diff
@@ -0,0 +1,61 @@
+Congratulations 2016 Graduates!
+-------------------------------
+:date: 2016-06-01
+:author: Amanda Kelner
+:slug: graduation-2016
+:img: OSCON7AlbertsonRettig.jpg
+
+As exciting as the end of the school year is for everyone, and with it the
+promise of sun, no classes, shorts, extended vacations, and no classes, it also
+brings an inevitable goodbye to those leaving to enter the real world.
+
+``*``sniffle``*``
+
+.. image:: /images/AnthonyMiller.jpg
+  :scale: 100%
+  :align: center
+  :alt: Anthony Miller, Community Systems Administrator
+
+I'm talking about our graduating seniors. Those who have earned their stripes
+and are ready to move on. the superstars stepping up to take the stage. The
+brave few willing to begin paying off their student loans. Many of our graduates
+have spent multiple years working for the OSL, some starting not long after high
+school. And we're losing too many good ones this year, including:
+
+.. image:: /images/MeganGossens.jpg
+  :scale: 100%
+  :align: left
+  :alt: Megan Gossens, Developer
+
+  **Megan Goosens-Developer**
+
+  **Anthony Miller-Community Systems Administrator**
+
+  **Elsie Phillips-Project Manager**
+
+.. image:: /images/ephillips.jpg
```

Comment:
Unless we're talking about Elsie's picture. That picture was already in the system because it's just her staff profile picture as we don't have any others of her.